src/style.css

```
@@ -1 +1,13 @@
 /* styles go here */
+
+.form-field {
+  margin-bottom: 10px;
+}
+
+.fieldset > .form-field:last-child {
+  margin-bottom: 0;
+}
```

Don't increase selector specificity.
```suggestion
.form-field:last-child {
  margin-bottom: 0;
}
```